README.md

```diff
@@ -1,59 +1,120 @@
 
-Scripts for building assets for both theme and blocks are located in the `package.json` file.
-Scripts for internationalization and Code Style are located in the `composer.json` file.
-In both files there are scripts that cover a wide range of functionalities use what you need.
+## Code Style & Quality
+
+This project uses PHP CodeSniffer for linting PHP files and the wrappers provided by `@wordpress/scripts` for linting JS/TS and CSS/SCSS files. You can find the scripts for linting and formatting PHP in the `composer.json` file and the scripts for linting and formatting JS/TS and CSS/SCSS in the `package.json` file.
+
+While JS/TS/CSS/SCSS linting should be configuration-free (using the defaults provided by `@wordpress/scripts`), the PHP linting requires a configuration file called `.phpcs.xml.dist` which is located in the root of the project. This file should never be modified (notice the `.dist` extension) without a good reason. Instead, you can create a new file called `.phpcs.xml` in the root of the project and add your customizations there although this is highly discouraged.
+
+Moreover, you will likely notice `.phpcs.xml` files sprinkled throughout the project (e.g., in the child theme and in the features plugin). These files are used to enhance the default configuration provided by the root `.phpcs.xml.dist` file for the files inside the respective folders. For example, they add checks for using the correct text domain for the theme and the features plugin, respectively, or for using the correct prefixes for global variables. Modifying these files is the preferred way to update the linting rules for the respective folders.
 
 ## Installing themes and plugins through Composer.json
 
-In composer.json there are examples on how to install a theme or plugin through composer.
-This can be used to install a parent theme or a plugin that is required for the theme to work.
+It's possible to install themes and plugins which are available on the WordPress.org repository through the `composer.json` file. This can help with development and debugging as it allows us to install a parent theme or a plugin that is required for the theme to work without tracking it through `git`.
+
+Installing non-WP.org themes and plugins via Composer is also possible if the plugin offers support for it. For example, all plugins sold through Freemius do support Composer, as well as some popular ones such as Advanced Custom Fields Pro.
 
-### Tips & Tricks (not required)
+### Installing WordPress itself as a Composer dependency
 
-You can include WordPress as a Composer dev dependency by including the package `johnpbloch/wordpress-core` as a Composer dev dependency.
+You can install WordPress itself as a Composer dependency by including the package `johnpbloch/wordpress-core`. This can be useful for PHPStorm's code completion or simply for searching through the codebase locally.
```

Comment:
Could we highlight that this is optional as is offered as a "nice to have" only.